api/config/v1/config.go

```diff
@@ -97,9 +97,13 @@ func DisableResourceNamingInConfig(logger logger, config *Config) {
 	config.Resources.MIGs = nil
 
 	// Disable renaming / device selection in Sharing.TimeSlicing.Resources
-	config.Sharing.TimeSlicing.disableResoureRenaming(logger, "timeSlicing")
+	if (config.sharing.TimeSlicing != nil) {
```

Comment:
Note: I think because we have:
```
func (rrs *ReplicatedResources) disableResoureRenaming(logger logger, id string) {
	if rrs == nil {
		return
	}
	...
```
we may not need the `nil` check.

Reply:
Thanks - removed in a separate commit